src/_locales/en/messages.json

```diff
@@ -405,21 +403,26 @@
 	"watched": {
 		"message": "Watched On"
 	},
-	"wrongItemDialogContent": {
+	"correctionDialogContent": {
```

Comment:
Perhaps you should sort this with new order now. (possible a job that can be automated)

Reply:
I have this for another PR as well.